### ISSUE TITLE
Receiving message stating that application is disabled

### ISSUE THREAD

**Issue by nameier** [EvanBasalik/OOFSponder#42]
Hello,

I started receiving the following this afternoon:

  
```
Request Id:  0d6cbb80-3b92-48e4-8899-72c9c1e65000 
 
Correlation Id:  3a5fb6c1-6026-4cc6-a205-22c25da8cddb 

Timestamp:  2024-04-01T18:01:26Z 
 
Message:  AADSTS7000112: Application 'c0eceb27-8cd3-4bb8-9271-c90596069f74'(OOFSponder) is disabled.
```

Exited and reloaded OOFSponder, and am still receiving the same error message.

**Comment by EvanBasalik** [comment]
Yeah - working to get it approved under the new policies, but no ETA.


From: nameier-MSFT ***@***.***>
Sent: Monday, April 1, 2024 2:03 PM
To: EvanBasalik/OOFSponder ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EvanBasalik/OOFSponder] Receiving message stating that application is disabled (Issue #42)


Hello,

I started receiving the following this afternoon:

Request Id:  0d6cbb80-3b92-48e4-8899-72c9c1e65000



Correlation Id:  3a5fb6c1-6026-4cc6-a205-22c25da8cddb



Timestamp:  2024-04-01T18:01:26Z



Message:  AADSTS7000112: Application 'c0eceb27-8cd3-4bb8-9271-c90596069f74'(OOFSponder) is disabled.

Exited and reloaded OOFSponder, and am still receiving the same error message.

-

**Comment by nameier** [comment]
[like]  Nathan Meier reacted to your message:
________________________________

**Comment by welersonlisboa1970** [comment]
Same problem is here,

**Comment by EvanBasalik** [comment]
Working to get it approved under the new policies, but no ETA.

**Comment by JM-007** [comment]
Thank you @EvanBasalik

**Comment by EvanBasalik** [comment]
OOFSponder has been reenabled.
